components/BottomNavigation/README.md

```diff
@@ -0,0 +1,183 @@
+<!--docs:
+title: "Bottom Navigation"
+layout: detail
+section: components
+excerpt: "Bottom navigation provides a bar at the bottom of the screen with items to navigate between different views."
+icon_id: bottom_navigation
+path: /catalog/bottomnavigation/
+api_doc_root: true
+-->
+
+# Bottom Navigation
+
+<div class="article__asset article__asset--screenshot">
+  <img src="docs/assets/bottomnavigation.png" alt="Bottom Navigation" width="375">
+</div>
+
+<a href="https://material.io/guidelines/components/bottom-navigation.html">Bottom navigation</a> makes it easy to explore and switch between top-level views in a single tap. Tapping on a bottom navigation icon takes you directly to the associated view or refreshes the currently active view.
+
+Bottom navigation should be used for: Three to five top-level destinations of similar importance. Destinations requiring direct access from anywhere in the app.
```

Comment:
I'd remove the `:` and the `3 to 5...` part. That's said with more detail in the next paragraph.